README.md

```diff
@@ -1,32 +1,41 @@
-# Apex CLI
+# Apexlang CLI
 
-Apex is an interface definition language (IDL) for modeling software. Generate
-source code, documentation, integration, everything automatically.
+The `apex` CLI is a one-stop shop for all projects across all languages.
 
-### Goals:
+It's a
 
-- <ins>A</ins>pproachable - Apex was designed from the ground up to be succinct.
-  Interfaces and data types are described using familiar syntax that won't slow
-  you down.
-- <ins>P</ins>rotocol agnostic - Regardless of the architecture, your data and
-  interfaces are fundamentally the same. Use Apex to generate code for any
-  serialization format or protocol.
-- <ins>Ex</ins>tensible - Generators are written in TypeScript. Easily add
-  custom generators that satisfy your unique needs and publish them for everyone
-  to use.
+- Project templating and scaffolding tool
+- Extensible code generation tool
+- Task runner
 
 For more information, visit [https://apexlang.io](https://apexlang.io).
 
-## Installation
+## Prerequisites
+
+The `apex` CLI depends on Deno.
 
-First, install [Deno](https://github.com/denoland/deno_install).
+Install `deno` with instructions
+[here](https://github.com/denoland/deno_install).
 
-Then run the command below from a terminal.
+## Installation
+
+To install a release version of the `apex` CLI, run the command below:
 
 ```
 deno install -A --unstable -f -n apex https://deno.land/x/apex_cli/apex.ts
 ```
 
+To install from source, clone this repository and run `./apex run install`
+
+```sh
+git clone git@github.com:apexlang/apex.git && cd apex && ./apex install
```

Comment:
I recommend changing the link to `https://github.com/apexlang/apex.git` because I think that is more commonly used.